File: week-08/day-02/List Todos/List Todos/Models/Todo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace List_Todos.Models
{
    public class Todo
    {
        public long Id { set; get; }
        public string Title { set; get; }
        public bool IsUrgent { set; get; }
        public bool IsDone { set; get; }
    }
}

File: week-11/Advanced backend/todoapp/src/main/java/com/todos/todoapp/Models/Todo.java
package com.todos.todoapp.Models;

import javax.persistence.*;

@Entity
@Table(name = "todos")
public class Todo {

  @Id
  @GeneratedValue(strategy = GenerationType.AUTO)
  private Long id;
  private String Title;
  private boolean isUrgent;
  private boolean isDone;

  public Long getId() {
    return id;
  }

  public void setId(Long id) {
    this.id = id;
  }

  public String getTitle() {
    return Title;
  }

  public void setTitle(String title) {
    Title = title;
  }

  public boolean isUrgent() {
    return isUrgent;
  }

  public void setUrgent(boolean urgent) {
    isUrgent = urgent;
  }

  public boolean isDone() {
    return isDone;
  }

  public void setDone(boolean done) {
    isDone = done;
  }
}
